Apps/WebClient/src/ClientApp/src/components/navmenu/HeaderComponent.vue

```diff
@@ -391,6 +419,7 @@ export default class HeaderComponent extends Vue {
         </b-navbar>
 
         <RatingComponent ref="ratingComponent" @on-close="processLogout()" />
+        <app-tour-component ref="appTourComponent" />
```

Comment:
Nit: we are moving away from kebab case for our components (particularly for ones that aren't in the shared folder and made available everywhere).  It's easier to track down references to the component when it's using the same name (e.g. `AppTourComponent`).